app/services/green/green-data.ts

```diff
@@ -0,0 +1,62 @@
+import { $t } from 'services/i18n';
+import { TPlatform } from 'services/platforms';
+
+export enum EGreenPlatform {
+  Twitch = 'twitch',
+  Facebook = 'facebook',
+  YouTube = 'youtube',
+  Trovo = 'trovo',
+}
```

Comment:
i'm not sure this is 100% needed since we have a type `TPlatform`, but either way if we do want an enum for platforms it should probably go there

Reply:
Moved. I'd like to use the enum so that I don't have to do any trickery trying to get the strings from `TPlatform`